test/StakeManager.js

```diff
@@ -255,7 +256,7 @@ describe('StakeManager', function () {
       await assertRevert(tx, 'Existing Lock doesnt exist');
     });
 
-    it('Staker should not be able to unstake more than his stoken balance', async function () {
+    it('Staker should not be able to unstake more than his sToken balance', async function () {
```

Comment:
Nomenclature: sRAZOR

Reply:
> Nomenclature: sRAZOR
what is sRAZOR balance ? Do you mean stake of staker ? @SkandaBhat

Reply:
What is sToken?